### DISCUSSION TITLE
4.14.0

### DISCUSSION THREAD

**Discussion by wimjongman** [eclipse-birt/birt#1510] [Announcements]
Hello all,

We have released 4.14 🎉

A big thank you 👏👏👏 to the main actors in this release:

Thomas Gutmann, alias [@speckyspooky](https://github.com/speckyspooky), for many quality patches  ⚒️ new features 🆕 and community support 💖
Ed Merks, alias [@merks](https://github.com/merks), for converting our old releng to [a spaceship](https://download.eclipse.org/birt/updates/release/latest/) 🛰️
Henning von Bargen alias [@hvbtup](https://github.com/hvbtup),  for quality patches  ⚒️ new features 🆕 and community support 💖
Claes Rossell, alias [@claesrosel](https://github.com/claesrosell), for always being prepared to fix the really hard integration problems 🧑‍🎓

On top of that, we have 4 new code contributors

*    [@Mailaender](https://github.com/Mailaender) made their first contribution in [#1232](https://github.com/eclipse-birt/birt/pull/1232)
*   [@vpease](https://github.com/vpease) made their first contribution in [#1233](https://github.com/eclipse-birt/birt/pull/1233)
*  [@94satya](https://github.com/94satya) made their first contribution in [#1429](https://github.com/eclipse-birt/birt/pull/1429)
*    [@slhotellier](https://github.com/slhotellier) made their first contribution in [#1222](https://github.com/eclipse-birt/birt/pull/1222)

Thanks to all who contributed in another way besides code patches. Your discussions and issue reports are invaluable.

Master is open for 4.15 development.

To the Batmobile, let's go!

Wim Jongman

**Comment by hvbtup** [comment]
I think it's worth to clarify in the release notes that BIRT 4.14 requires Java 17 or higher.

**Reply by wimjongman** [reply]
Thanks, I added it: https://github.com/eclipse-birt/birt/releases/tag/4.14.0

**Reply by MikeDel001** [reply]
I am currently running birt 4.8 on java 8. obviously i cant run 4.14 but, what is the highest version of birt i can run?

**Reply by merks** [reply]
You have 1.3 years before Java 8 EOL.   You might want to use that remaining time window to move forward...

**Reply by MikeDel001** [reply]
Thanks merks. Any idea what the highest version is i can run on 8?

**Reply by claesrosell** [reply]
The last version that works with Java 8 is Birt 4.8.
When the project restarted, with Birt 4.9, we moved to Java11.

**Reply by MikeDel001** [reply]
Perfect, thank you for your help.
